services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -32,39 +32,26 @@ const propTypes = {
 };
 
 const defaultProps = {
-  sortField: null,
-  sortDir: null,
+  sortField: undefined,
+  sortDir: undefined,
   noticeOfWorkApplications: [],
-  searchParams: {},
 };
 
-const handleTableChange = (updateApplicationList) => (pagination, { mine_region } = {}, sorter) => {
-  const sortParams = isEmpty(sorter)
-    ? {
-        sort_field: undefined,
-        sort_dir: undefined,
-      }
-    : {
-        sort_field: sorter.column.sortField,
-        sort_dir: sorter.order.replace("end", ""),
-      };
-  const params = isEmpty(mine_region)
-    ? {
-        ...sortParams,
-        mine_region: undefined,
-      }
-    : {
-        ...sortParams,
-        mine_region,
-      };
-
+const handleTableChange = (updateApplicationList) => (pagination, filters, sorter) => {
```

Comment:
Thanks and great job fixing the table behaviors! 🥳

Reply:
No problem! It was fun looking into it, but as we discussed, I'll hold off on doing anything else to them until we decide on how we want to present the filtering capabilities (e.g., the Mine Lookup table page provides an alternative method for filtering the results).